Fix - Run only the "native" tests, not on the rp240

test/test_native/test_main.cpp:10: test_native_environment	[PASSED]
---------------- native:test_native [PASSED] Took 14.64 seconds ----------------

=================================== SUMMARY ===================================
Environment        Test                   Status    Duration
-----------------  ---------------------  --------  ------------
seeed_xiao_rp2040  test_protocol_handler  ERRORED   00:02:06.304
seeed_xiao_rp2040  test_simple            ERRORED   00:00:05.281
native             test_native            PASSED    00:00:14.642
================== 3 test cases: 1 succeeded in 00:02:26.227 ==================
Error: Process completed with exit code 1.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -23,4 +23,4 @@ jobs:
         run: pip install --upgrade platformio
 
       - name: Test PlatformIO Project
-        run: pio test --without-uploading
+        run: pio test -e native

